aldryn_people/admin.py

```diff
@@ -17,24 +17,31 @@ class PersonAdmin(AllTranslationsMixin, TranslatableAdmin):
     list_display = [
         '__str__', 'email', 'vcard_enabled', ]
     list_filter = ['groups', 'vcard_enabled']
-    search_fields = ('name', 'email', 'translations__function')
-    prepopulated_fields = {'slug': ('name',)}
+    search_fields = ('translations__name', 'email', 'translations__function')
     raw_id_fields = ('user',)
 
+    def get_prepopulated_fields(self, request, obj=None):
```

Comment:
Good point. However, prepopulated slug fields will be removed in the next PR =) Thanks.